sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -1033,7 +1029,7 @@ private void ThrowLockLostException()
             {
                 amqpRequestMessage.AmqpMessage.ApplicationProperties.Map[ManagementConstants.Request.AssociatedLinkName] = receiveLink.Name;
             }
-            amqpRequestMessage.Map[ManagementConstants.Properties.LockTokens] = new[] { new Guid(lockToken) };
+            amqpRequestMessage.Map[ManagementConstants.Properties.LockToken] = lockToken;
```

Comment:
Nice :)

Reply:
Wait, does this work? @danielmarbach I thought you said the service throws when trying this.

Reply:
No I had reverted that locally. It slipped in. Weird I thought I forced push the undo. Let me check

Reply:
Pushed again. Sorry for that. I had the undo locally but reset it when changing branches 🤦‍♂️

Reply:
@pakrym sorry for making you excited for nothing 🤗

Reply:
![](https://media4.giphy.com/media/Q3Ou3eFKMTFn2/giphy.gif)